cucumber/FreeEnrollmentPlan.feature

```diff
@@ -0,0 +1,70 @@
+@i9n
+	Feature: configuração de assinatura gratuita
+
+	As a administrador de curso
+	I want to disponibilizar cursos de forma gratuita
+	So that os usuários podem consumir conteúdo sem custo o conteúdo do curso
```

Comment:
So that os usuários podem consumir conteúdo sem custo o conteúdo do curso =>
So that os usuários podem consumir o conteúdo do curso sem custo